ffi/tests/ffi_export.rs

```diff
@@ -199,6 +201,78 @@ pub fn freestanding_with_repr_c_struct(
     struct_
 }
 
+/// Return `u128`
+#[ffi_export]
+pub fn freestanding_u128(value: u128) -> u128 {
```

Comment:
I would put all these into a separate file `ffi_export_u128`. Do we need import tests?

Reply:
Better safe than sorry i guess, so i will add them.

Reply:
I know that tests look horrible and unorganized for ffi. I was hoping we can instill some order now and then